app/lib/actions/katello/repository/sync_deb_errata.rb

```diff
@@ -0,0 +1,95 @@
+module Actions
+  module Katello
+    module Repository
+      class SyncDebErrata < Actions::Base
+        def plan(repo)
+          plan_self(repo_id: repo.id)
+        end
+
+        # rubocop:disable MethodLength
+        # rubocop:disable AbcSize
+        def run
+          if SETTINGS[:katello][:cdn_proxy] && SETTINGS[:katello][:cdn_proxy][:host]
```

Comment:
I think we have a better way of handling proxy usage. See https://github.com/Katello/katello/pull/8181

Reply:
Thanks for the tip, looks much nicer now.